test/e2e/scalability/load.go

```diff
@@ -535,8 +535,21 @@ func GenerateConfigsForGroup(
 			MemRequest:     26214400, // 25MB
 			SecretNames:    secretNames,
 			ConfigMapNames: configMapNames,
-			// Define a label to group every 2 RCs into one service.
-			Labels: map[string]string{svcLabelKey: groupName + "-" + strconv.Itoa((i+1)/2)},
+		}
+
+		// We assume by default that every RC is part of its own service.
+		// However, you can override it using RC_TO_SERVICE_RATIO env var.
+		// For simplicity, we only allow integer values for the env var.
+		rcToServiceRatio := 1
+		if rcToServiceRatioEnv := os.Getenv("RC_TO_SERVICE_RATIO"); rcToServiceRatioEnv != "" {
```

Comment:
I just thought about it and I'm not convinced that this is the best way of doing that.
Let's say that I would like a service for 2/3rd of services - with this approach it's not possible.

I think we need something slightly more generic that 1/n-th.

Reply:
I know this sucks :)
Also exactly the reason why I added that comment `For simplicity, we only allow integer values for the env var`. But if you want I can try to enable floating point service:RC ratio. Does that sgty?

Reply:
My fear is that you will see that with 1/2 of backends everything will work and then you're stuck because with this code you can't really test e.g. 2/3rd backends.
So I think that floating point ration makes more sense here.

Reply:
Makes sense. Changed it, PTAL.